griptape/tools/rest_api_client/tool.py

```diff
@@ -10,13 +10,32 @@
 
 @define
 class RestApiClient(BaseTool):
-    base_url: str = field(kw_only=True)
-    path: str = field(default="", kw_only=True)
-    description: Optional[str] = field(default=None, kw_only=True)
-    request_path_params_schema: Optional[str] = field(default=None, kw_only=True)
-    request_query_params_schema: Optional[str] = field(default=None, kw_only=True)
-    request_body_schema: Optional[str] = field(default=None, kw_only=True)
-    response_body_schema: Optional[str] = field(default=None, kw_only=True)
+    base_url: str = field(
+        factory=str,
+    )
+    path: Optional[str] = field(
+        default=None,
+        kw_only=True,
+    )
+    description: str = field(
+        factory=str,
+    )
```

Comment:
Same issue here: this will generate the default value of `""`. Should either be an optional with a Jinja conditional in the activity decorator code below or not have a default/factory value defined here.